Makefile

```diff
@@ -147,17 +148,17 @@ deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in
 undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/config. Call with ignore-not-found=true to ignore resource not found errors during deletion.
 	$(KUSTOMIZE) build config/default | kubectl delete --ignore-not-found=$(ignore-not-found) -f -
 
-CONTROLLER_GEN = controller-gen
+CONTROLLER_GEN = $(shell pwd)/bin/controller-gen
```

Comment:
this is basically a autogenerated one by the kubebuilder, why do we need this change?

Reply:
Actually, hitting this error due to incorrect PATH
```
bash: line 1: controller-gen: command not found
make: *** [Makefile:96: generate] Error 127
```

Reply:
Would it be fine if I export PATH in the beginning of Makefile ?

```
export PATH:=$(shell pwd)/bin:$(PATH)
```